docs/cyber.md

```diff
@@ -0,0 +1,80 @@
+# access anomalies: [complement_access.py](../src/main/python/mmlspark/cyber/ml/access_anomalies/complement_access.py)
+- [Talk at European Spark Conference 2019](https://databricks.com/session_eu19/cybermltoolkit-anomaly-detection-as-a-scalable-generic-service-over-apache-spark)
+- [(Internal Microsoft) Talk at MLADS November 2018](https://resnet.microsoft.com/video/42395)
```

Comment:
I don't think the videos are publicly accessible.

Reply:
The first one is public, it was given in the European Spark Summit late 2019. The other two are only internally available in MS

Reply:
hmm. I'd personally not post internal only links to GH, but open to suggestion.

Reply:
Is there a way to have these visible to MS users (internally) and hidden for external open source users?